source/_components/sensor.filter.markdown

```diff
@@ -97,9 +97,9 @@ LowPass(state) = A * previous_state + B * state
 
 The returned value is rounded to the number of decimals defined in (`precision`).
 
-### {% linkable_title Outlier %}
+### {% linkable_title Band-pass %}
 
-The Outlier filter (`outlier`) is a basic Band-stop filter, as it cuts out any value outside a specific range.
+The Band-pass filter (`outlier`) cuts out any value outside a specific range.
```

Comment:
The filter is `outlier`. So, let us not rename the filter. Feel free to update the description.